.github/scripts/cache.py

```diff
@@ -41,16 +41,16 @@
 
 PWD = os.getcwd()
 
-GITHUB_ORG="bittide"
-GITHUB_REPO="bittide-hardware"
+GITHUB_ORG = "bittide"
+GITHUB_REPO = "bittide-hardware"
```

Comment:
Could you please open another PR adding a formatter? You'll need to add it to `nix/git-hooks.nix`. The most popular one is `black` AFAIK.